frontend/src/main/components/Driver/DriverShiftsTable.js

```diff
@@ -0,0 +1,34 @@
+import OurTable from "main/components/OurTable"
+
+
+export default function DriverShiftsTable({ drivershifts }) {
+
+    const columns = [
+        {
+            Header: 'id',
+            accessor: 'id', // accessor is the "key" in the data
+        },
+        {
+            Header: 'Day',
+            accessor: 'day',
+        },
+        {
+            Header: 'Shift start',
+            accessor: 'shiftStart',
+        },
+        {
+            Header: 'Shift end',
+            accessor: 'shiftEnd',
+        },
+        {
+            Header: 'Backup driver',
+            accessor: 'driverBackupID',
+        }
+    ];
+
```

Comment:
Define this outside the component to not ruin table's shallow equality optimizations.
```suggestion
const columns = [
    {
        Header: 'id',
        accessor: 'id', // accessor is the "key" in the data
    },
    {
        Header: 'Day',
        accessor: 'day',
    },
    {
        Header: 'Shift start',
        accessor: 'shiftStart',
    },
    {
        Header: 'Shift end',
        accessor: 'shiftEnd',
    },
    {
        Header: 'Backup driver',
        accessor: 'driverBackupID',
    }
];

export default function DriverShiftsTable({ drivershifts }) {
```